src/FTPClient.jl

```diff
@@ -16,6 +16,7 @@ type FTPClientError <: Exception
        lib_curl_error::UInt32
 end
 
+@enum FTP_MODE ascii_mode binary_mode
```

Comment:
I would prefer `@enum FTP_MODE ascii binary` still but you can leave it with `_mode` for this PR.